retrieval/resource_map.go

```diff
@@ -131,10 +131,22 @@ var ProxyV2ResourceMap = ResourceMap{
 	},
 }
 
+var AnthosL4LBMap = ResourceMap{
```

Comment:
Recommend adding `MatchLabel`. But it implied that you need to add the corresponding labels/annotation you specified for `MatchLabel`. An example is [ProxyV2](https://github.com/Stackdriver/stackdriver-prometheus-sidecar/blob/c8c0bfb1a5e22f5838eb6bb86608b29ef0eca0ef/retrieval/resource_map.go#L120-L132).

The motivation behind `MatchLabel` is to give option for the kubernetes users (not the sidecar maintainers) that if their `LabelMap` all match, but the users do not want to export the metrics with the monitored resource, the users just don't add the `MatchLabel` to their workload - and these metrics can be still exported as other monitored resource (e.g. `k8s_container`). 

Though I would say that the labels you set up are really specific and users are not likely to have conflict with your labels, I would still recommend adding `MatchLabel` in the first place; but we can discuss if there are other concerns.

Reply:
Thanks for your advice, didn't think about the edge case before. I've updated the code correspondingly.